frontend/talentsearch/src/js/components/Router.tsx

```diff
@@ -22,28 +21,65 @@ import {
   DirectIntakeRoutes,
   useDirectIntakeRoutes,
 } from "../directIntakeRoutes";
-import RequestPage from "./request/RequestPage";
-import WorkLocationPreferenceApi from "./workLocationPreferenceForm/WorkLocationPreferenceForm";
-import { ProfilePage } from "./profile/ProfilePage/ProfilePage";
-import ExperienceFormContainer from "./experienceForm/ExperienceForm";
 import { ExperienceType } from "./experienceForm/types";
-import WorkPreferencesApi from "./workPreferencesForm/WorkPreferencesForm";
-import { GovInfoFormContainer } from "./GovernmentInfoForm/GovernmentInfoForm";
-import LanguageInformationFormContainer from "./languageInformationForm/LanguageInformationForm";
-import AboutMeFormContainer from "./aboutMeForm/AboutMeForm";
-import DiversityEquityInclusionFormApi from "./diversityEquityInclusion/DiversityEquityInclusionForm";
-import { ExperienceAndSkillsRouterApi } from "./applicantProfile/ExperienceAndSkills";
-import RoleSalaryFormContainer from "./roleSalaryForm/RoleSalaryForm";
-import BrowsePoolsPage from "./browse/BrowsePoolsPage";
-import BrowseIndividualPoolApi from "./browse/BrowseIndividualPool";
-import PoolApplyPage from "./pool/PoolApplyPage";
-import PoolApplicationThanksPage from "./pool/PoolApplicationThanksPage";
-import RegisterPage from "./register/RegisterPage";
-import LoggedOutPage from "./loggedOut/LoggedOutPage";
-import LoginPage from "./login/LoginPage";
-import { CreateAccount } from "./createAccount/CreateAccountPage";
 import { Role } from "../api/generated";
-import PoolAdvertisementPage from "./pool/PoolAdvertisementPage";
+
+/** Search */
+const SearchPage = React.lazy(() => import("./search/SearchPage"));
```

Comment:
I observed a cypress failure to load a chunk during a local run though I'm not sure why.  When I opened my browser interactively it didn't have a problem.  It was successful when I ran it a second time, too.  Is this going to increase the number of random cypress failures?  Can we mitigate this somehow?
![image](https://user-images.githubusercontent.com/8978655/180785422-5a0adef1-d137-4301-9f2d-007f34c5b933.png)